src/Upgrade/Infrastructure/PackageManager/ComposerAdapter.php

```diff
@@ -104,6 +103,16 @@ public function getComposerLockFile(): array
         return $this->composerLockReader->read();
     }
 
+    /**
+     * @param \Upgrade\Domain\Entity\Collection\PackageCollection $packageCollection
+     *
+     * @return \Upgrade\Application\Dto\PackageManagerResponseDto
+     */
+    public function updateSubPackages(PackageCollection $packageCollection): PackageManagerResponseDto
```

Comment:
Please, cover this method with tests.
The same for the `isSubPackage` method.

Reply:
https://github.com/spryker-sdk/upgrader/pull/165/files#diff-b9071e2f72caa248abe1005edbc8d166f770f8cfaa50e30cfaeb91dfdc08418eR115